db/seeds.d/openscap_policy_notification.rb

```diff
@@ -0,0 +1,9 @@
+policy_notification = {
+  :name => :openscap_policy_summary,
+  :description => N_('A summary of reports for OpenScap policies'),
+  :mailer => 'ForemanOpenscap::PolicyMailer',
+  :method => 'policy_summary',
+  :subscription_type => 'report',
+}
+
+MailNotification.first_or_create(policy_notification)
```

Comment:
This is a wrong syntax, the mail notification will not be created.
The syntax should be:
```ruby
MailNotification.where(policy_notification).first_or_create
```